### ISSUE TITLE
Functions are not printed when they are attributes of objects

### ISSUE THREAD

**Issue by tdb-alcorn** [tdb-alcorn/nice-logger#1]
```
var obj = {x: 1, y: "foo", z: function() { return true; }};
log.info(obj);
>> {"x": 1, "y": "foo"}
```
This occurs because the logger defaults to calling JSON.stringify on anything that isn't a String or an Error, and JSON.stringify does not serialize functions because it is intended as a wire serialization format.

Add an object stringifier (that makes an effort to print everything, including functions), and call it in a new case for handling objects.

**Comment by tdb-alcorn** [comment]
In the above example, acceptable output would be something like
```
>> {"x": 1, "y", "foo", "z", [Function]}
```

**Comment by tdb-alcorn** [comment]
Fixed in 7b64858.
